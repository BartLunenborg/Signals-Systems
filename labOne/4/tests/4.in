100 100 100 15 0.1
