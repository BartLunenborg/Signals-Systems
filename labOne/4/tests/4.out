[99,50,-40,-97,-74,9,86,91,21,-66,-99,-50,40,97,74,-9,-86,-91,-21,66,99,50,-40,-97,-74,9,86,91,21,-66,-99,-50,40,97,74,-9,-86,-91,-21,66,99,50,-40,-97,-74,9,86,91,21,-66,-99,-50,40,97,74,-9,-86,-91,-21,66,99,50,-40,-97,-74,9,86,91,21,-66,-99,-50,40,97,74,-9,-86,-91,-21,66,99,50,-40,-97,-74,9,86,91,21,-66,-99,-50,40,97,74,-9,-86,-91,-21,66]
